packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts

```diff
@@ -54,6 +54,10 @@ export function isAzureProject(azureSettings: AzureSolutionSettings | undefined)
   return azureSettings !== undefined && HostTypeOptionAzure.id === azureSettings.hostType;
 }
 
+export function isVsPlatform(ctx: v2.Context): boolean {
```

Comment:
@hund030  Just curious if there is any common place to put this isVsPlatform() logic? I am not sure if this method exist in both solution, tab and bot plugin?

Reply:
Yes. Maybe we can put this in common/tools.ts.

Reply:
Good idea！

Reply:
we already have this function globally: 
![image](https://user-images.githubusercontent.com/1658418/154036968-6e5d79f0-eb9c-43bc-bdcc-9f04c603252e.png)

Reply:
Thanks for sharing @jayzhang, @JerryYangKai can you help check if we can leverage this?